bluepysnap/config.py

```diff
@@ -37,32 +39,34 @@
     "edges_type_file",
     "nodes_type_file",
     "node_sets_file",
+    "output_dir",
+    "network",
+    "mechanisms_dir",
```

Comment:
This is probably not relevant to this PR:
If we also consider the ngv keys (I noticed vasculature is here too), there are a couple missing, such as:
* microdomains_file
* endfeet_meshes_file (this should replace the 'end_feet_areas')